lib/node_modules/@stdlib/math/base/special/bernoulli/benchmark/c/native/benchmark.c

```diff
@@ -91,15 +91,18 @@ static double rand_double( void ) {
 */
 static double benchmark( void ) {
 	double elapsed;
-	double x;
+	double x[ 100 ];
 	double y;
 	double t;
 	int i;
 
+	for ( i = 0; i < 100; i++ ) {
+		x[ i ] = ( 500.0 * rand_double() );
+	}
+
 	t = tic();
 	for ( i = 0; i < ITERATIONS; i++ ) {
-		x = ( 500.0 * rand_double() );
-		y = stdlib_base_bernoulli( x );
+		y = stdlib_base_bernoulli( x[ i % 100 ] );
```

Comment:
```suggestion
		y = stdlib_base_bernoulli( (int)( x[ i % 100 ] ) );
```

Making the input argument cast more explicit.